src/styles.scss

```diff
@@ -0,0 +1,59 @@
+/* You can add global styles to this file, and also import other style files */
+
+@import '../node_modules/angular-archwizard/archwizard';
+
+html, body {
+  margin: 0 auto;
+  padding: 0 auto;
+  height: 100%;
+  width: 100%;
+}
+
+
+// Define styles for the wizard on the 'Free navigation mode' demo page.
+// For this wizard (selected by a custom CSS class `.free-navigation`) we adjust
+// the styling of wizard steps so that optional steps look like default ones.
+
+@mixin free-navigation-wizard() {
```

Comment:
Is it also possible to provide this style definition inside a `free-navigation-mode.component.scss` file?

Reply:
Yes, but one will need something like [::ng-deep](https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep) in order for styles to "reach" the internals of the nested wizard component.

Reply:
Are you sure? The custom css example(s) don't need `ng-deep` to work, see also [custom-global-css.component.css](https://github.com/madoar/angular-archwizard-demo/blob/develop/src/app/custom-global-css/custom-global-css.component.css).

Reply:
You are right, it will work if component uses `encapsulation: ViewEncapsulation.None`.

Reply:
Should I change this?

Reply:
I think it is better to set the style as near as possible to example. This has the benefits that:
- the user doesn't need to navigate far to find the styles
- the style changes are less likely to influence the styles of other examples

Reply:
Hmm... It looks as if angular-archwizard needs to expose SCSS variables and mixins in a separate file.  

Currently application code can only use these variables and mixins if it imports the whole 'archwizard.scss'. But it cannot be done from without a component styles file, because it will duplicate the styles for archwizard.

I will create a separate pull request..

Reply:
MR for angular-archwizard:
https://github.com/madoar/angular-archwizard/pull/217